modules/serverless-functions-all-values-in-configmap-to-env-variables.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-accessing-secrets-configmaps.adoc
```

Comment:
Sorry @rh-max one last thing - can you please remove all these "Module included in..." statements, we're not required to do this anymore 🙂